code/pizzabot.js

```diff
@@ -1,7 +1,86 @@
+
+// Pizza sorts:
+
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+console.log("Vegetarian Pizza", "Hawaiian Pizza","Pepperoni Pizza");
+
+
+
+//iteration 2// 
+//alert(`Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}`);
+
+//iteration 3 ,4 and 5// 
+/* const orderName = prompt("Enter the name of the pizza you want to order today.");
+if ((orderName === vegetarian) || (orderName === hawaiian) || (orderName === pepperoni)) {
+  const orderQuantity = prompt(`How many ${orderName}(s) do you want?`);
+  const orderTotal = orderQuantity * pizzaPrice;
+    if (orderQuantity <= 2) {
+      alert(`Great, I'll get started on your ${orderName} pizza(s) right away, it will cost ${orderTotal} kr. The pizza(s) will take 10 minutes.`);
+    } else if (orderQuantity > 2 && orderQuantity <= 5) {
+      alert(`Great, I'll get started on your ${orderName} pizza(s) right away, it will cost ${orderTotal} kr. The pizza(s) will take 15 minutes.`);
+    } else {
+      alert(`Great, I'll get started on your ${orderName} pizza(s) right away, it will cost ${orderTotal} kr. The pizza(s) will take 20 minutes.`);
+    }
+} else {
+  alert('Select a pizza from the menu');
+}
+*/
```

Comment:
This code is really good for keeping track of how you got to where you are.  It's a good idea to remove it before submitting it for review, though.  

As long as you commit often, you can see this code in the commit history. For instance, https://github.com/Technigo/project-pizza-bot/pull/24/commits/fdd71aa6abbd69636a5ba0908294cdb9f3a60307